src/UserService.Business/Commands/Certificate/CreateCertificateCommand.cs

```diff
@@ -5,10 +5,13 @@
 using LT.DigitalOffice.Kernel.Exceptions.Models;
```

Comment:
Wait. I did it here https://github.com/LT-Students/DigitalOffice-UserService/pull/121

Reply:
I edited this because of updated version of models doesn't contain IAddImageResponse which was here